### PR TITLE
Feature/form element input text

### PR DESCRIPTION
**Pull request by ronaldo-mh**

### CODE REVIEW

**Comment by jcger** [comment]
Is this going to be in any production env soon? (same for the textarea)

I mean, UX is working on this, maybe it would be possible to coordinate it with them

File: components/form/textInput/src/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const FormTextInput = ({className, errorMessage, name, onChange, placeholder, value}) => (
+  <div className={cx('sui-FormTextInput', className)}>
+    <input
+      className='sui-FormTextInput-value'
+      name={name}
+      type='text'
+      placeholder={placeholder}
+      value={value}
+      onChange={onChange} />
+    {renderErrorMessage(errorMessage)}
```
Comment (JoanClaret) [inline components/form/textInput/src/index.js]: What do you think about checking if there is an error before to execute the function?
`error && renderErrorMessage(errorMessage)`

File: components/form/textInput/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    &:focus {
+      outline-width: 0;
+    }
+  }
+}
+
+.sui-FormTextInput {
+  &-value {
+    @include sui-text-input;
```
Comment (JoanClaret) [inline components/form/textInput/src/index.scss]: i don't get the point of having a mixin here. 
does we pass `$default-border` or `$default-border` somewhere?

File: components/form/textInput/src/index.js (hunk 1 above)
Comment (ronaldo-mh) [reply components/form/textInput/src/index.js]: Yes, it is better on that way. I will change it in next commit! Thanks! 👍

File: components/form/textInput/src/index.scss (hunk 2 above)
Comment (ronaldo-mh) [reply components/form/textInput/src/index.scss]: Yes, there is not point as the default borders won't be used anymore.
I will change it in next commit.

File: components/form/textInput/src/index.js (hunk 1 above)
Comment (JoanClaret) [reply components/form/textInput/src/index.js]: Great!

File: components/form/textInput/src/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const renderErrorMessage = error => (
+  error && (
+    <div className='sui-FormTextInput-errorMessage'>
+      <span className='sui-FormTextInput-errorMessageLabel'>{error}</span>
+    </div>
+  )
+)
+
+const FormTextInput = ({className, errorMessage, name, onChange, placeholder, value}) => (
```
Comment (jcger) [inline components/form/textInput/src/index.js]: You are adding this class in your other PR too, please remove wherever it has to be removed

Comment (ronaldo-mh) [reply components/form/textInput/src/index.js]: Yes, this is the right branch for these files, I have just removed from the other PR, thanks!

**Comment by ronaldo-mh** [comment]
@julian-gernun @JoanClaret 
- Removed the unused mixin.
- Removed wrong files from PR (corresponding to textarea's PR).
- Added comment in README.md to clarify the "provisional" status of this component.

Please review! Thanks!

**Review by JoanClaret — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Pull request merged**